tools/astarte_e2e/lib/astarte_e2e/config.ex

```diff
@@ -113,6 +114,18 @@ defmodule AstarteE2E.Config do
     type: :pos_integer,
     default: 10
 
+  @envdoc "The comma-separated email recipients. The default value is not a valid address and it must be overridden."
+  app_env :mailer_to_address, :astarte_e2e, :mailer_to_address,
+    os_env: "ASTARTE_E2E_MAILER_TO_ADDRESS",
+    type: ListOfStrings,
+    default: "test@mail-example.com"
```

Comment:
If the default is invalid doesn't it make more sense to omit the default and add `required: true`?

Reply:
This choice prevents (again) the ci from failing as, in that situation, we don't actually need to specify a mail recipient.
Either we set the recipient among the env vars for the ci, or we accept an invalid default in here.